app/models/participant_profile/ecf.rb

```diff
@@ -39,3 +39,6 @@ def fundable?
     ecf_participant_eligibility&.eligible_status?
   end
 end
+
+require_dependency "participant_profile/ect"
```

Comment:
nit: should be at the top of the file

Reply:
@jamesrose intermediate class need to be fully loaded before loading its descendant, otherwise intermediate/parent classes won't know they have descendants, it's all explained in the guides:

https://guides.rubyonrails.org/autoloading_and_reloading_constants_classic_mode.html#autoloading-and-sti